test/rulesets/Random/random.jl

```diff
@@ -19,4 +19,27 @@
             @test all(map(x -> x isa Zero, pb(10)))
         end
     end
+
+    @testset "rand" begin
+        non_differentiables = [((), Float64),
+                               ((Random.default_rng(),), Float64),
+                               ((Random.default_rng(),2,2), Matrix{<:Float64}),
+                               ((Float32,), Float32),
+                               ((Float32,2,2), Matrix{<:Float32}),
+                               ((Float32,(2,2)), Matrix{<:Float32}),
+                               ((2,2), Matrix{<:Float64})]
+
+        for (args,xType) in non_differentiables
+            x, dΩ = frule((), rand, args...)
```

Comment:
Lets make it a realistic call to `frule`
```suggestion
            x, dΩ = frule((Zero(), 1.2.*ones(args)...), rand, args...)
```